### PR TITLE
[envTest] Enable webhooks in envTest and update the DefaultContainerImage test

### PR DESCRIPTION
**Pull request by fmount**
This change improves the existing `envTest` functional tests starting the `webhooks` within the `suite_test` setup and fixing the `ContainerImage` test accordingly. 
In addition, as done for `dataplane-operator` [1], this patch improves the `Make` target using `ginkgo` cli, while `go test` is still used to run regular unit tests.
This is very useful for development purposes, where we would like to focus on that single test that is going to be written and get a more verbose log, as well as being able to customize concurrency and other fields.

```
e.g.

make test GINKGO_ARGS="-v --focus \
  "'Should fail if db-sync job fails when DB is Created'"
```

Other than that, it's easier to run tests in parallel and produce reports [2].

[1] openstack-k8s-operators/dataplane-operator#204
[2] https://onsi.github.io/ginkgo/#the-ginkgo-cli-vs-go-test

### CODE REVIEW

**Comment by fmount** [comment]
CI seems happy [1] and I see `ginkgo` is now used.

```
Will run 22 of 22 specs
Running in parallel across 7 processes
••••••••••••••••••••••

Ran 22 of 22 Specs in 13.649 seconds
SUCCESS! -- 22 Passed | 0 Failed | 0 Pending | 0 Skipped

coverage: 100.0% of statements
composite coverage: 100.0% of statements

Ginkgo ran 1 suite in 18.26953674s
Test Suite Passed
```

[1] https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openstack-k8s-operators_manila-operator/77/pull-ci-openstack-k8s-operators-manila-operator-main-functional/1665618142321184768/artifacts/test/build-log.txt

File: tests/functional/suite_test.go [hunk 1]
```diff
@@ -167,8 +186,13 @@ var _ = BeforeSuite(func() {
 		Kclient: kclient,
 		Log:     ctrl.Log.WithName("controllers").WithName("Manila"),
 	}).SetupWithManager(k8sManager)
+
+	Expect(err).NotTo(HaveOccurred())
 	Expect(err).ToNot(HaveOccurred())
```
Comment (gibizer) [inline tests/functional/suite_test.go:191]: you can drop one of these

**Review by gibizer — APPROVED** [review]
Nicely done!

File: tests/functional/suite_test.go (hunk 1 above)
Comment (fmount) [reply tests/functional/suite_test.go:191]: oh yeah, they're basically the same thing :D

**Review by gibizer — APPROVED** [review]

**Comment by gibizer** [comment]
Seems to me we have an unstable test

**Pull request merged**